spec/javascripts/resources/InventoryParticipantSpec.js

```diff
@@ -0,0 +1,21 @@
+describe('Resource: InventoryParticipant', function() {
+  var subject, scope, $httpBackend;
+
+  beforeEach(module('PDRClient'));
+
+  beforeEach(inject(function($resource, $rootScope, InventoryParticipant, $injector) {
+    scope = $rootScope.$new();
+    subject = InventoryParticipant;
+    $httpBackend = $injector.get('$httpBackend');
+  }));
+
+  it("creates participant", function(){
```

Comment:
Similar to the previous resource test, this seems to be the right approach, but what you want to do is rename this expectation.  We would expect that the create method is called with the parameters we expect it to.